src/Section/HarvestsSection/HarvestsSection.js

```diff
@@ -23,16 +24,39 @@ class HarvestsSection extends Component {
       }
   }
 
+  getGraphData() {
+    let data = {}
+    for (var i = this.state.harvests.length - 1; i >= 0; i--) {
+      const harvest = this.state.harvests[i]
+      if (harvest.status === "successful") {
+        const date = new Date(harvest.finished).toLocaleDateString().split('-').reverse().join('/')
+        data[date] = harvest.itemsFound
```

Comment:
I have not been not very inspired. This is a table containing the chart data of crops...

Why not `crops`?